.phpstan/phpstan.neon.dist

```diff
@@ -14,3 +13,5 @@ parameters:
         - stubs/MetadataInterface.file
         - stubs/Exporter.file
         - stubs/TextExtension.file
+    # NEXT_MAJOR: remove next line. It was added because in v3.x types was mostly on PhpDoc level
+    # treatPhpDocTypesAsCertain: false
```

Comment:
is those 2 lines needed?

Reply:
no need, I was doing with it first, but than thought that I need it to be more strict